.github/workflows/make-release.yml

```diff
@@ -12,6 +12,9 @@ jobs:
   finish_release:
     if: github.event.pull_request.merged == true && startsWith(github.head_ref, 'release/automated_v')
     runs-on: ubuntu-latest
+    environment:
+      name: Production
+      url: ${{ steps.publish-github-release.outputs.html_url }}
```

Comment:
I'm not sure if this variable will be available at the beginning of the run, or if it is evaluated after the run... we shall see :)

Reply:
This is evaluated after the run